README.md

```diff
@@ -69,3 +69,16 @@ You can upload to any WebDAV server which supports `PUT` operations. The followi
 - `$WEBDAV_RELEASE_NAME`: name of the release directory (optional on *GitHub actions*)
 
 **Note:** Secrets must not be stored inside the repository, nor be visible to end users. You need to store them securely, ideally using the credentials storage your build system provides (on GitHub actions, there's *Secrets*, for instance).
+
+
+## Changelog Generation
+`pyuploadtool` support Changelog generation, which is optional, and can be enabled with the `$CHANGELOG_GENERATE` environment variable.
+```bash
+CHANGELOG_GENERATE=true ./pyuploadtool*.AppImage
+```
+### Changelog Types
+`pyuploadtool` currently supports two types of Changelogs.
```

Comment:
> Wouldn't it be easier to just have one variable for everything? How big are the chances that CHANGELOG_TYPE is set without the other variable?

Yea, that's good. And that also helps by not hardcoding a default for commit style.

> Also, I'd like to avoid numeric values, and I'd also go for lowercase values and uppercase keys.

Fixed both of these in https://github.com/TheAssassin/pyuploadtool/pull/3/commits/50455a8328ffaa4a822819c1a6a8574c70887d5a and https://github.com/TheAssassin/pyuploadtool/pull/3/commits/6e18de8e35b1bf3f012b1251c74fd071a4bb7328